gcloud/_helpers.py

```diff
@@ -439,8 +441,7 @@ def _to_bytes(value, encoding='ascii'):
     :rtype: str / bytes
     :returns: The original value converted to bytes (if unicode) or as passed
               in if it started out as bytes.
-    :raises: :class:`TypeError <exceptions.TypeError>` if the value
-             could not be converted to bytes.
+    :raises TypeError: if the value could not be converted to bytes.
```

Comment:
Yeah totally. For now I'm not touching `missing-raises-doc` in this PR other than the 3 or 4 places I modified it to understand what was triggering the error / how to fix it.